procedures/unit-testing-autorun.ipf

```diff
@@ -69,17 +69,23 @@ static Function AfterFileOpenHook(refNum, file, pathName, type, creator, kind)
 			catch
 				err = GetRTError(1)
 				print "The run() function aborted with an RTE and this can not be handled."
+				QuitOnAutoRunFull()
```

Comment:
@MichaelHuth Why quit here and not at line 80 before `End`?

Reply:
@MichaelHuth `cmd` is now unused.

Reply:
If I move the Quit to the end then it changes functionality. The idea is that by default Igor quits (for AUTORUN_FULL) when RunTest finishes. However in the case that run() throws an RTE RunTest could be never called, though Igor should still quit.

When I move it to the end like:
```
if(err)
  QuitOnAutoRunFull()
endif
```

Would work too, but it splits the error handling code to too separate locations.

Reply:
@MichaelHuth Good point!